cmd/calculator.go

```diff
@@ -30,26 +30,25 @@ func init() {
 	cmdCalculator.Flags().BoolVar(&calculatorOpts.awsDebug, "aws-debug", false, "Log debug information from aws-sdk-go library")
 }
 
-func runCmdCalculator(_ *cobra.Command, _ []string) error {
+func runCmdCalculator(_ *cobra.Command, _ []string) {
 
 	opts := root.NewOptions(false, false)
 
 	cluster, err := root.ClusterFromFile(configPath, opts, calculatorOpts.awsDebug)
 	if err != nil {
-		return fmt.Errorf("Failed to initialize cluster driver: %v", err)
+		logger.Fatalf("Failed to initialize cluster driver: %v", err)
```

Comment:
Can we retain `return`s for command failures? I prefer it over `logger.Fatalf` because then the golang compiler can enforce that we don't miss any error handling.

Reply:
yeah, sure

Reply:
Thanks. You're always responsive!

Reply:
ok, done, the only problem is that cobra errors won't be in color. Let me know what you think

Reply:
Good point! Hopefully we could find a way to gain goods of both. How about wrapping the cobra commands with a wrapper that calls `log.Fatalf` on error?